### PR TITLE
Public trades and tickers config

### PR DESCRIPTION
**Pull request by SaenkoM**
Adds tickers section to snapshots
Adds `amountUsd` column to movements
Separates configuration for `public trades` and `tickers`
Separates start time for `public trades`, `public funding` and `tickers`
Fixes configs symbol mapping
Fixes tooltip on sync pref apply button
Adds translations
Adds default start time for `account balance` and `win/loss` sections
Improves `concentration risk` section

### CODE REVIEW

File: src/components/ConcentrationRisk/ConcentrationRisk.js [hunk 1]
```diff
@@ -40,20 +41,28 @@ class ConcentrationRisk extends PureComponent {
...
+
+      return {
+        currency: key,
+        balanceUsd,
+        percent: fixedFloat(percent),
+      }
+    }), 'balanceUsd').reverse()
 
-    const chartData = tableData.map(({ currency, balanceUsd }) => ({
+    const chartData = tableData.filter(({ percent }) => percent > 0.1).map(({ currency, balanceUsd }) => ({
```
Comment (jarindr) [inline src/components/ConcentrationRisk/ConcentrationRisk.js]: can we drop the chain in 2 lines?
.filter
.map
for readability

Comment (SaenkoM) [reply src/components/ConcentrationRisk/ConcentrationRisk.js]: @jarindr changed :)

File: src/state/sync/saga.js [hunk 2]
```diff
@@ -251,7 +251,7 @@ function* initSync() {
         progress: syncProgress,
       }))
     } else {
-      yield call(startSyncing)
+      // yield call(startSyncing)
```
Comment (ezewer) [inline src/state/sync/saga.js]: Please remember to remove this comment

File: src/components/Snapshots/Snapshots.js [hunk 3]
```diff
@@ -118,7 +120,8 @@ class Snapshots extends PureComponent {
     const section = this.getCurrentSection()
     const hasNewTime = timestamp ? currentTime !== timestamp.getTime() : !!currentTime !== !!timestamp
 
-    const isNotEmpty = !!(positionsEntries.length || tickersEntries.length || walletsEntries.length)
+    const isNotEmpty = !!(positionsEntries.length || positionsTickersEntries.length
+      || walletsTickersEntries || walletsEntries.length)
```
Comment (jarindr) [inline src/components/Snapshots/Snapshots.js]: is `walletsTickersEntries` an array ?

File: src/components/Snapshots/Snapshots.js [hunk 4]
```diff
@@ -192,7 +195,7 @@ class Snapshots extends PureComponent {
         <Loading title='snapshots.title' />
       )
     } else if ((section === MENU_POSITIONS && !positionsEntries.length)
-      || (section === MENU_TICKERS && !tickersEntries.length)
+      || (section === MENU_TICKERS && (!positionsTickersEntries.length && !walletsTickersEntries))
```
Comment (jarindr) [inline src/components/Snapshots/Snapshots.js]: let's move this check as a variable, I think kinda too long here XD

File: src/components/Snapshots/Snapshots.js (hunk 3 above)
Comment (ezewer) [reply src/components/Snapshots/Snapshots.js]: yes

**Pull request merged**